library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -32,11 +36,15 @@ import org.xmtp.proto.keystore.api.v1.Keystore.TopicMap.TopicData
 import org.xmtp.proto.message.contents.Contact
 import org.xmtp.proto.message.contents.Invitation
 import org.xmtp.android.library.messages.DecryptedMessage
+import uniffi.xmtpv3.FfiConversations
+import uniffi.xmtpv3.FfiListConversationsOptions
+import uniffi.xmtpv3.org.xmtp.android.library.libxmtp.GroupEmitter
 import java.util.Date
 
 data class Conversations(
     var client: Client,
     var conversationsByTopic: MutableMap<String, Conversation> = mutableMapOf(),
+    private val libXMTPConversations: FfiConversations? = null,
```

Comment:
Technically I could get this from the client via `client.v3Client.conversations` But I kind of liked passing it explicitly. I could go either way.